cmake/thirdparty/SetupChaiThirdparty.cmake

```diff
@@ -42,6 +42,7 @@
 #######################################################################
 
 message (STATUS "cnmem support is ${ENABLE_CNMEM}")
+message (STATUS "pick/set/incr/decr support is ${ENABLE_PICK_SET_INCR_DECR}")
```

Comment:
This message shouldn't be here, this is for third-party library stuff. Just delete the message.